source/expressions.tex

```diff
@@ -6118,7 +6118,7 @@
 \begin{itemize}
 \item
 If one pointer represents the address of a complete object, and another
-pointer represents the address one past the last element of a different
+pointer represents the address past the end\iref{basic.compound} of a different
```

Comment:
This is about the represented address, not the pointer value, so I think this change isn't what we want. Something like this might work, but it's a bit clumsy: "and another pointer represents the same address as a pointer past the end of a different complete object".

Reply:
How about "pointer represents the address of a pointer value past the end of a different complete object"

Reply:
I think we need "same address" somewhere in the mix.